pom.xml

```diff
@@ -257,6 +257,16 @@
             <artifactId>quartz</artifactId>
             <version>2.3.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.papertrailapp</groupId>
+            <artifactId>logback-syslog4j</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.janino</groupId>
```

Comment:
Ah, Janino was needed for the conditional Papertrail appender in the logback.xml. We used that to avoid having to write the same logic as part of a custom appender.